ui/pages/Search/components/VariantSearchFormPanels.jsx

```diff
@@ -340,12 +441,16 @@ class VariantSearchFormPanels extends React.PureComponent {
         </ExpandCollapseCategoryContainer>
         <VerticalSpacer height={10} />
         <Accordion fluid exclusive={false}>
-          {panels.reduce((acc, { name, headerProps, ...panelContentProps }, i) => {
+          {PANELS.reduce((acc, { name, headerProps, fields, ...panelContentProps }, i) => {
```

Comment:
this actual code change to more clearly set the panels instead of requiring a fully configured constant panel array to be passed as a prop